### PR TITLE
(DOCSP-17258): Release Notes, 07/02/2021

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-17258

### Staged Changes (Requires MongoDB Corp SSO)

- [Release Notes](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-17258/release-notes/backend/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/release-notes/changelogs/backend/changelog-backend-2021.rst [hunk 1]
```diff
@@ -1,3 +1,13 @@
+.. _backend_20210702:
+
+July 02 2021 Release
```
Comment (dacharyc) [inline source/release-notes/changelogs/backend/changelog-backend-2021.rst:3]: Tiny nit: this date format is different than the last one (16 June 2021 vs. July 02 2021) - perhaps it should be consistent?

**Review by dacharyc — APPROVED** [review]
LGTM with a tiny nit about date format.

File: source/release-notes/changelogs/backend/changelog-backend-2021.rst (hunk 1 above)
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2021.rst:3]:
```suggestion
02 July 2021 Release
```

Comment (nathan-contino-mongo) [reply source/release-notes/changelogs/backend/changelog-backend-2021.rst]: Good catch, that's a huge mistake on my part. Thanks!

**Review by dacharyc — APPROVED** [review]
LGTM!

**Comment by sumedham** [comment]
ready to merge as of this release - https://mongodb.slack.com/archives/CMMLKL5SB/p1625236203032400

**Pull request merged**